sql/plan.go

```diff
@@ -152,7 +152,7 @@ func (p *planner) makePlan(stmt parser.Statement) (planNode, error) {
 	}
 }
 
-func (p *planner) query(sql string) (planNode, error) {
+func (p *planner) query(sql string, args parser.Args) (planNode, error) {
```

Comment:
Since this is only used internally, perhaps change `args` to `args ...interface{}`. This would simplify usage by the caller and more closely mimic the `sql/database.DB.{Query,QueryRow,Exec}` interface.